mlir/test/fusion/tosa-tp2-highlevel-fused.mlir

```diff
@@ -1,7 +1,7 @@
 // RUN: mlir-miopen-driver --host-pipeline highlevel %s | FileCheck %s
 
-// CHECK: miopen.conv2d(%{{.*}}, %{{.*}}, %{{.*}}) features =  none {arch = "gfx906", dilations = [1 : i32, 1 : i32], filter_layout = ["k", "c", "y", "x", "g"], input_layout = ["ni", "hi", "wi", "ci", "gi"], numCu = 64 : i32, output_layout = ["no", "ho", "wo", "ko", "go"], padding = [1 : i32, 1 : i32, 1 : i32, 1 : i32], strides = [1 : i32, 1 : i32]} : memref<64x128x3x3x1xf32>, memref<256x28x28x128x1xf32>, memref<256x28x28x64x1xf32>
-    
+// CHECK: miopen.conv2d(%{{.*}}, %{{.*}}, %{{.*}}) features =  none {arch = "gfx900", dilations = [1 : i32, 1 : i32], filter_layout = ["k", "c", "y", "x", "g"], input_layout = ["ni", "hi", "wi", "ci", "gi"], numCu = 64 : i32, output_layout = ["no", "ho", "wo", "ko", "go"], padding = [1 : i32, 1 : i32, 1 : i32, 1 : i32], strides = [1 : i32, 1 : i32]} : memref<64x128x3x3x1xf32>, memref<256x28x28x128x1xf32>, memref<256x28x28x64x1xf32>
```

Comment:
BTW, I would prefer not to see 'features = none', just like I don't see 'attributes {}'. Can you hide it when none.

Reply:
I don't think so, given that `features` is a required attribute and that `none` is how you print out its value when it's 0. And if we weren't doing a custom thing like that we'd have things like `features = 1 : i32` for having `mfma` on.